### PR TITLE
Add info endpoints to asgt dataservice

### PR DESCRIPTION
**Pull request by lyschoening**
Refactor DatasetInfo and ModelInfo to Dataset and Model

### CODE REVIEW

File: proto/asgt/dataservice/v1alpha1/electronic_invoice_line_data_service.proto [hunk 1]
```diff
@@ -32,6 +33,13 @@ service ElectronicInvoiceLineDataService {
       body: "*"
     };
   }
+
+  rpc GetBankInfo (GetInfoRequest) returns (GetInfoResponse) {
```
Comment (JacobSMoller) [inline proto/asgt/dataservice/v1alpha1/electronic_invoice_line_data_service.proto:37]: Copy paste error

File: proto/asgt/dataservice/v1alpha1/scanned_invoice_data_service.proto [hunk 2]
```diff
@@ -32,6 +33,13 @@ service ScannedInvoiceDataService {
       body: "*"
     };
   }
+
+  rpc GetBankInfo (GetInfoRequest) returns (GetInfoResponse) {
```
Comment (JacobSMoller) [inline proto/asgt/dataservice/v1alpha1/scanned_invoice_data_service.proto:37]: Copy paste error

File: proto/asgt/dataservice/v1alpha1/bq_entries.proto [hunk 3]
```diff
@@ -24,7 +24,7 @@ message BankEntry {
   repeated string tags = 4 [(gen_bq_schema.bigquery).description = "Tags defined by consumer, enriched by service if valetkey is used"];
   asgt.type.BankTransaction data = 5;
   repeated asgt.type.TargetValue target_values = 6;
-  asgt.type.ModelInfo model = 9;
+  asgt.type.Model model = 9;
```
Comment (JacobSMoller) [inline proto/asgt/dataservice/v1alpha1/bq_entries.proto:27]: Do we store the model info in BQ?

Comment (lyschoening) [reply proto/asgt/dataservice/v1alpha1/bq_entries.proto:27]: Yes, currently we store the version number. This would add the created_at date too.

Comment (JacobSMoller) [reply proto/asgt/dataservice/v1alpha1/bq_entries.proto:27]: :+1:

**Pull request merged**